sygnal/http.py

```diff
@@ -201,6 +202,28 @@ async def cb():
             if not root_span_accounted_for:
                 root_span.finish()
 
+    def find_pushkins(self, appid):
+        """
+        Finds matching pushkins in self.sygnal.pushkins according to the appid.
+        If finds a specific pushkin with the exact app id, immediately returns
+        it (as a list).
+        Otherwise returns possible pushkins (as a list).
+
+        appid: app identifier to search in self.sygnal.pushkins.
```

Comment:
We're usually using Google's docstring style for arguments and such: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html (e.g. https://github.com/matrix-org/synapse/blob/54283f3ed49ce40059e9f0746f2ded08ce3a5b9e/synapse/util/threepids.py#L22-L32)

What this means is that arguments should look like:

```
Args:
    arg1 (type1): short description
    arg2 (type2): short description
```

We haven't always been good at always remembering to mention the type, and it's not mandatory, but most Python IDEs support it and it helps them to show accurate docs and do some better type checking/linting. Could you rewrite that part of the docstring to match this style please?

Reply:
Sure 👍